x-pack/plugins/ingest_manager/server/services/epm/packages/install.ts

```diff
@@ -372,7 +312,13 @@ async function installPackage({
       install_source: installSource,
     });
   }
-  const installIndexPatternPromise = installIndexPatterns(savedObjectsClient, pkgName, pkgVersion);
+  // TODO without doing an await here I get an unhandled promise rejection
+  const installIndexPatternPromise = await installIndexPatterns(
```

Comment:
@jfsiii It think this was the call that was causing the unhandled promise rejection when I ran the registry with the injected failure.

Reply:
That sounds right. We need a to deal with any rejection from `installIndexPatterns`. We can use `await` in front or add a `.catch` on the end.